### PR TITLE
Release 136.0.0

### PR DESCRIPTION
**Pull request by NEllusion**
## Explanation

Releases version `9.1.0` of the `@metamask/controller-utils` package.

### CODE REVIEW

File: packages/controller-utils/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [9.0.3]
+
+### Added
```
Comment (Gudahtt) [inline packages/controller-utils/CHANGELOG.md:12]: Nit: I would call this a bugfix, so "Fixed" seems like a more appropriate category. That matches the version bump as well; we typically use a minor bump for features/enhancements, but a patch for bug fixes.

File: packages/controller-utils/CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [9.0.3]
```
Comment (Gudahtt) [inline packages/controller-utils/CHANGELOG.md:10]: It looks like there is a change missing from this changelog: https://github.com/MetaMask/core/pull/4030

It wasn't mentioned in the PR description, but that PR adds a new constant.

Comment (Gudahtt) [reply packages/controller-utils/CHANGELOG.md:10]: Probably a good idea to mention this as a change as well: https://github.com/MetaMask/core/pull/4084

We don't typically include dev dependency updates in changelogs, but a TypeScript bump does change the output in ways that may be important for consumers (e.g. if there was a bug fix/enhancement to `tsc` that impacts the compiled JS).

Comment (Gudahtt) [reply packages/controller-utils/CHANGELOG.md:10]: Given the addition of that constant, a minor bump seems more appropriate for this release I think

**Review by Gudahtt — APPROVED** [review]
LGTM!

**Review by Mrtenz — APPROVED** [review]

**Pull request merged**